packages/common/api.js

```diff
@@ -198,8 +198,10 @@ export const getUserItems = () => callApi.get('/items/all');
 export const getItem = itemId => callApi.get(`/items/${itemId}`);
 export const getItemRaws = itemId => callApi.get(`/items/${itemId}/raws`);
 export const postCreateItem = data => callApi.post('/items', data);
-export const getRemovedItems = itemsIds =>
-  callApi.post('/items/unexists', { items: itemsIds });
+export const getRemovedItems = (itemsIds, teamId) =>
+  callApi.post(`/items/unexists${teamId ? `?team=${teamId}` : ''}`, {
+    items: itemsIds,
```

Comment:
You can use the params: 
```
{
items: itemsIds,
teamId
}
```
if the teamId will null or empty, then the teamId would be remove

Reply:
Replaced with teamId 👍